deploy/storage/bigquery.version

```diff
@@ -1 +1 @@
-google.com:datcom-store-dev.dc_kg_2021_03_23_14_53_54
\ No newline at end of file
+	google.com:datcom-store-dev.dc_kg_2021_03_26_01_01_15
```

Comment:
Looks like there's extra whitespaces before the version string. Hope the consumers aren't finicky, like they are about not needing a trailing \n. Might be worth fixing...

Reply:
ok, I will fix that, good catch!